ensime_shared/client.py

```diff
@@ -232,14 +219,13 @@ def disable_completely(e):
                 port = self.ensime.http_port()
                 uri = "websocket" if server_v2 else "jerky"
                 self.ensime_server = gconfig["ensime_server"].format(port, uri)
-            with catch(Exception, disable_completely):
-                from websocket import create_connection
+            with catch(websocket.WebSocketException, disable_completely):
```

Comment:
Bonus benefit: we can catch properly scoped exceptions now in several places, this one should only be a `WebSocketException` and not also `ImportError`.